types/index.test-d.ts

```diff
@@ -21,6 +21,7 @@ const expectResponse = function (res: Response) {
   expectAssignable<Function>(res.json)
   expectAssignable<http.ServerResponse>(res.raw.res)
   expectAssignable<http.IncomingMessage>(res.raw.req)
+  expectType<Array<Date | undefined>>(res.cookies.map(x => x.expires))
```

Comment:
As this is a PR to improve the typings,we should test intensively the typings:

```suggestion
  expectAssignable<Array<any>>(res.cookies)
  expectType<string>(res.cookies[0].name)
  expectType<string>(res.cookies[0].value)
  expectType<Date | undefined>(res.cookies[0].expires)
  expectType<number | undefined>(res.cookies[0].maxAge)
  expectType<boolean | undefined>(res.cookies[0].httpOnly)
  expectType<boolean | undefined>(res.cookies[0].secure)
  expectType<string | undefined>(res.cookies[0].sameSite)
  expectType<unknown | undefined>(res.cookies[0].additional)
```

Reply:
I assumed tsd would enable `noUncheckedIndexedAccess` so I want to avoid using `.cookies[0]`, but now looks like it doesn't ...